packages/pwa-kit-create-app/assets/templates/@salesforce/retail-react-app/app/components/_app-config/index.jsx.hbs

```diff
@@ -46,6 +46,9 @@ const AppConfig = ({children, locals = {}}) => {
 
     const appOrigin = getAppOrigin()
 
+    const redirectURI = `${appOrigin}/callback`
+    const proxy = `${appOrigin}${getEnvBasePath()}${commerceApiConfig.proxyPath}`
```

Comment:
If I was a partner dev and I saw this line of code I would probably be quite confused. Since we are giving people this file, this code is theirs and they should probably be able to understand it. 

Have you thought about either creating a new util called `getAppOriginWithBasePatth` or maybe modifying so we can optionally include the base path? like `getAppOrigin({includeBasePath: true})` 

Just a thought.

Reply:
It's a good thought, but I think we may need to put more effort into this. I'd really love to get this merged soon and move on to the second part of this work...